code/modules/surgery/surgery.dm

```diff
@@ -44,15 +48,22 @@
 	if(replaced_by == /datum/surgery)
 		return FALSE
 
-	if(HAS_TRAIT(user, TRAIT_SURGEON) || (user.mind && HAS_TRAIT(user.mind, TRAIT_SURGEON)))
+	if(HAS_TRAIT(user, TRAIT_SURGEON) || user.mind && HAS_TRAIT(user.mind, TRAIT_SURGEON))
```

Comment:
Please keep good bracketisation of boolean conditionals, it makes it clear what the intention is since it is significantly less clear of the precedence of these.

Reply:
Ah. As mentioned in my other PR, I wasn't aware that was the proper way to go about it. Fixed